libs/base/docs/reference/text/to-text.md

```diff
@@ -1,7 +1,7 @@
-# to string
+# to text
 
 Converts a value into a text representation.
 
 ```sig
-toString(123)
+toText(123)
```

Comment:
to match the blocks, it should be "convertToText"

Reply:
good call, switched